bot/src/database/services/rawInfo.service.ts

```
@@ -4,6 +1,172 @@
+            } else {
+                after = fetchedUsers[fetchedUsers.length - 1].id
+            }
+        } else if (response.status === 429) {
+            const rateLimitInfo = await response.json()
+            const retryAfter = rateLimitInfo.retry_after * 1000
+            logger.warn(
+                { channel_id: channelId, message_id: messageId },
+                `Rate limited for emoji: ${encodedEmoji}. Retrying after ${rateLimitInfo.retry_after} seconds.`
+            )
+            await new Promise((resolve) => setTimeout(resolve, retryAfter))
+            continue
```

Biome (1.9.4)

[error] Unnecessary continue statement

Unsafe fix: Delete the unnecessary continue statement


(lint/correctness/noUnnecessaryContinue)